testing/run-atomic-tests.ps1

```diff
@@ -0,0 +1,20 @@
+IEX (New-Object Net.WebClient).DownloadString('https://raw.githubusercontent.com/redcanaryco/atomic-red-team/master/execution-frameworks/Invoke-AtomicRedTeam/install-atomicredteam.ps1'); Install-AtomicRedTeam -verbose

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Invoke-Expression is used. Please remove Invoke-Expression from script and find other options instead.](https://app.codacy.com/manual/ION28/BLUESPAWN/pullRequest?prid=5092780)